desktop/bin/build-desktop-secrets.js

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env node
+
+/* eslint-disable no-console*/
+/* eslint-disable no-process-exit*/
+/* eslint-disable import/no-nodejs-modules*/
+
+const path = require( 'path' );
+const { execSync } = require( 'child_process' );
+
+const CALYPSO_SECRETS_ENCRYPTION_KEY = process.env.CALYPSO_SECRETS_ENCRYPTION_KEY;
+if ( ! CALYPSO_SECRETS_ENCRYPTION_KEY ) {
+	console.error( 'Failed to decrypt: CALYPSO_SECRETS_ENCRYPTION_KEY is not set.' );
```

Comment:
If I get here how will I know how to set it? Can you give more direction on what to do?